packages/docs/site/i18n/gu/docusaurus-plugin-content-docs/current/developers/05-local-development/01-wp-now.md

```diff
@@ -0,0 +1,114 @@
+---
+title: wp-now
+slug: /developers/local-development/wp-now
+---
+
+:::caution પેકેજ ડિપ્રિકેટેડ (deprecated) છે
+NPM પેકેજ @wp-now/wp-now ડિપ્રિકેટેડ (deprecated) છે, ભવિષ્યમાં અપડેટ્સ પ્રાપ્ત થશે નહીં. તમારા ડેવલપર ફ્લો પર કમાન્ડ-લાઇન ટૂલનો ઉપયોગ કરવા માટે, NPM પેકેજ `@wp-playground/cli` નો ઉપયોગ કરો.
+:::
+
+<!--
+:::caution Package deprecated
+The NPM package @wp-now/wp-now is deprecated, won't receive updates in the future. To use a command-line tool on your developer flow, use the NPM package `@wp-playground/cli`.
+:::
+-->
+
+# wp-now NPM પેકેજ
+
+<!--
+# wp-now NPM package
+-->
+
+[wp-now](https://www.npmjs.com/package/@wp-now/wp-now) એ એક કમાન્ડ-લાઇન ટૂલ છે જે સ્થાનિક રીતે વર્ડપ્રેસ ચલાવવાની પ્રક્રિયાને સરળ બનાવવા માટે રચાયેલ છે. તે ન્યૂનતમ રૂપરેખાંકન સાથે સ્થાનિક વર્ડપ્રેસ વાતાવરણ સેટ કરવાની ઝડપી અને સરળ રીત પ્રદાન કરે છે.
+
+<!--
+[wp-now](https://www.npmjs.com/package/@wp-now/wp-now) is a command-line tool designed to simplify the process of running WordPress locally. It provides a quick and easy way to set up a local WordPress environment with minimal configuration.
+-->
+
+મુખ્ય વિશેષતાઓ:
+
+<!--
+Key Features:
+-->
+
+-  **કમાન્ડ-લાઇન ઇન્ટરફેસ**: CLI સાથે આરામદાયક વિકાસકર્તાઓ માટે ઉપયોગમાં સરળ.
+-  **ઝડપી સેટઅપ**: સેકન્ડોમાં સ્થાનિક વર્ડપ્રેસ વાતાવરણ સેટ કરો.
+-  **કસ્ટમાઇઝેબલ**: ચોક્કસ વિકાસ જરૂરિયાતોને અનુરૂપ ગોઠવણી માટે પરવાનગી આપે છે.
+
+<!--
+-   **Command-line Interface**: Easy to use for developers comfortable with CLI.
+-   **Quick Setup**: Set up a local WordPress environment in seconds.
+-   **Customizable**: Allows for configuration to suit specific development needs.
+-->
+
+[`@wp-now/wp-now`](https://www.npmjs.com/package/@wp-now/wp-now) એ એક CLI ટૂલ છે જે એક જ આદેશથી વર્ડપ્રેસ સાઇટને સ્પિન કરે છે. [VS કોડ એક્સટેન્શન](/developers/local-development/vscode-extension) ની જેમ, તે PHP અને SQLite ના પોર્ટેબલ WebAssembly વર્ઝનનો ઉપયોગ કરે છે. કોઈ ડોકર, MySQL, અથવા Apache ની જરૂર નથી.
+
+<!--
+[`@wp-now/wp-now`](https://www.npmjs.com/package/@wp-now/wp-now) is a CLI tool to spin up a WordPress site with a single command. Similarly to the [VS Code extension](/developers/local-development/vscode-extension), it uses a portable WebAssembly version of PHP and SQLite. No Docker, MySQL, or Apache are required.
+-->
+
+:::info **દસ્તાવેજીકરણ**
+
+`wp-now` એક અલગ GitHub રિપોઝીટરી, [પ્લેગ્રાઉન્ડ ટૂલ્સ] (https://github.com/WordPress/playground-tools/) માં જાળવવામાં આવે છે. તમે નવીનતમ દસ્તાવેજીકરણ [dedicated README ફાઇલ] (https://github.com/WordPress/playground-tools/blob/trunk/packages/wp-now/README.md) માં શોધી શકો છો.
```

Comment:
I think the white space between the label of the link can generate a invalid link `[પ્લેગ્રાઉન્ડ ટૂલ્સ] (https://github.com/WordPress/playground-tools/)`, please remove the space `[પ્લેગ્રાઉન્ડ ટૂલ્સ](https://github.com/WordPress/playground-tools/)` to create a valid link.

Reply:
Also, `[dedicated README ફાઇલ] (https://github.com/WordPress/playground-tools/blob/trunk/packages/wp-now/README.md)` should we translate dedicated to this case?